src/elements/content-preview/preview-header/PreviewHeader.js

```diff
@@ -68,7 +68,7 @@ const PreviewHeader = ({
     });
 
     return (
-        <div className={className}>
+        <div data-testid="previewheader" className={className}>
```

Comment:
Any thoughts on keeping this the same as the class (`bcpr-header`), at least initially?